x/genutil/client/cli/init.go

```diff
@@ -81,14 +80,6 @@ func InitCmd(mbm module.BasicManager, defaultNodeHome string) *cobra.Command {
 
 			serverCtx := server.GetServerContextFromCmd(cmd)
 			config := serverCtx.Config
-
-			// default mamaki configs TODO: move to the app to be more explicit
-			// in what we are setting as defaults
-			config.Consensus.TimeoutCommit = time.Second * 25
-			config.Consensus.SkipTimeoutCommit = false
-			config.Mempool.KeepInvalidTxsInCache = true
```

Comment:
not copying this to celestia-app per https://github.com/celestiaorg/celestia-app/issues/590#issuecomment-1281468138